client/components/course/Settings/index.vue

```diff
@@ -57,7 +57,10 @@ export default {
       showCloneModal: false
     };
   },
-  computed: mapGetters(['course', 'outlineActivities'], 'course'),
+  computed: {
+    ...mapGetters(['course', 'outlineActivities'], 'course'),
+    publishPercentage: ({ publishStatus }) => publishStatus.progress * 100
```

Comment:
Why not convert it to percentage upfront?

Reply:
That's actually my fault. Because `[0..1]` is _agnostic_ range that allows you to easily transpose to other potential ranges required by particular component. Yeah in our case we deal with `v-loader` only so assuming `[0-100]` is safe bet but still 😁